discover/portal/articles/04-business-productivity/01-forms/00-forms-intro.markdown

```diff
@@ -2,16 +2,16 @@
 
 Collecting information from your site's Users is important, whether you're
 asking them to submit a brief survey or apply for a mortgage. In either case,
-you need to design a form. Liferay gives you almost limitless form building
-capability with the Forms application.
+you need a form. Liferay gives you almost limitless form building capability
+with the Forms application.
```

Comment:
Hey @rbohl, in the Forms and Lists intro, you've referenced the Forms Portlet as Liferay Forms, but here you're calling it as just Forms, thus I think that would be nice to include a note or just a sentence in the intro telling the reader that the Forms is known as Liferay Forms too. I'm raising this concern, because as we've several forms, the user can make confusion and be in doubt if we want the refer to the Forms application or all Liferay Forms, for example.

What do you think about that?

Reply:
@natocesarrego I agree. Maybe I'll just use "Liferay Forms" here too. I'll look at it in context and see what makes sense.